project-management/src/main/java/life/qbic/projectmanagement/application/authorization/QbicOidcUser.java

```diff
@@ -73,10 +69,19 @@ public String getName() {
   }
 
   public String getOidcId() {
-    return originalAuthName;
+    return Optional.ofNullable(originalAuthName).orElse(qbicUserInfo.oidc());
+  }
+
+  public String getOidcIssuer() {
+    return Optional.ofNullable(super.getIssuer().toString()).orElse(qbicUserInfo.oidcIssuer());
```

Comment:
Is it safe to load it from the database? I say no as the information should come only from the DefaultOidcUser filled by the auth flow.